.github/ISSUE_TEMPLATE/documentation_update.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Documentation update
+about: Create a report to help us improve potku-console documentation
+title: "[Documentation update]"
+labels: documentation
+assignees: softgitron, EddieTheCubeHead, joomakelastudent
+
+---
+
+**Describe the supposed change**
+A clear and concise description of the proposed change
+
+**Pitch your addition with a user story**
+As a \[role\] I can \[capability\], so that \[receive benefit\]
```

Comment:
I think this is slightly overkill here, but I think, this could be good in the documentation addition.

Reply:
Should be fixed in new pr